packages/model/src/event.ts

```diff
@@ -4,6 +4,7 @@ import { ListResponse } from './common';
 
 export const MEETING_LINK_AVAILABLE_HOURS_BEFORE_EVENT: number = 24;
 export const EVENT_CONSIDERED_PAST_HOURS_AFTER_EVENT: number = 1;
+export const EVENT_CONSIDERED_IN_PROGRESS_BEFORE_MINUTES: number = 5;
```

Comment:
```suggestion
export const EVENT_CONSIDERED_IN_PROGRESS_MINUTES_BEFORE_EVENT: number = 5;
```
for consistent naming